src/main/java/htw/kbe/maumau/domain/MauMau.java

```diff
@@ -0,0 +1,74 @@
+package htw.kbe.maumau.domain;
+
+import java.util.List;
+
+public interface MauMau {
+
+    /**
+     * Draws the given amount of cards
+     * @param amount amount of cards that will get drawn
+     * @return list of cards that will be drawn
+     */
+    List<Card> drawCards(int amount);
```

Comment:
```suggestion
    List<Card> initialCardDealing(int amount);
```

Bei der Benennung der Methode sollte es sofort klar sein, dass es sich um die einmalige Kartenausgabe zum Start des Spiels handelt.

Reply:
Die Methode ist aber nicht nur zum initial verteilen der Karten sondern auch zum ziehen von x Karten wenn eine 7 auf eine 7 gelegt wird oder nur eine sieben. Dann zieht man 2 oder eben 4 Karten. Man kann aber auch 5 karten ziehen zum beginn des spiels.

Reply:
Das stimmt nicht, dafür habt ihr extra eine neue Methode geschrieben im Interface >> mustDraw()

Reply:
Ne das sollte eigentlich keine Methode sondern ein Attribut vom Spieler sein.

Reply:
Ne sorry mein Fehler. Hatte die vergessen das wir das am ende hinzugefügt hatten. sorry :D